docs/source/ml.rst

```diff
@@ -39,23 +39,40 @@ values in a column. The "sparse_*" models include a LASSO regression step
+Name                              Model Type          Algorithm                                                                                                                            Altered Defaults
+================================  ================    ==================================================================================================================================   ==================================
+sparse_logistic                   classification      `LogisticRegression <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html>`_                ``C=499999950, tol=1e-08``
+gradient_boosting_classifier      classification      `GradientBoostingClassifier <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html>`_    ``n_estimators=500, max_depth=2``
+random_forest_classifier          classification      `RandomForestClassifier <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html>`_            ``n_estimators=500``
+extra_trees_classifier            classification      `ExtraTreesClassifier <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesClassifier.html>`_                ``n_estimators=500``
+multilayer_perceptron_classifier  classification      `MLPClassifier <https://github.com/civisanalytics/muffnn>`_ 
+stacking_classifier               classification      `StackedClassifier <GET_LINK_FROM_CIVISMLEXT>`_ 
+sparse_linear_regressor           regression          `LinearRegression <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LinearRegression.html>`_ 
+sparse_ridge_regressor            regression          `Ridge <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html>`_ 
+gradient_boosting_regressor       regression          `GradientBoostingRegressor <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html>`_      ``n_estimators=500, max_depth=2``
+random_forest_regressor           regression          `RandomForestRegressor <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html>`_              ``n_estimators=500``
+extra_trees_regressor             regression          `ExtraTreesRegressor <http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.ExtraTreesRegressor.html>`_                  ``n_estimators=500``
+multilayer_perceptron_regressor   regression          `MLPRegressor <https://github.com/civisanalytics/muffnn>`_ 
+stacking_regressor                regression          `StackedRegressor <GET_LINK_FROM_CIVISMLEXT>`_ 
+================================  ================    ==================================================================================================================================   ==================================
+
+The "stacking_classifier" model stacks
```

Comment:
How does this render? This line length looks a little short?

Reply:
When it builds, single line breaks don't matter, so this doesn't affect the rendering.